tests/pages/logout.py

```diff
@@ -0,0 +1,15 @@
+from selenium.webdriver.common.by import By
+
+from .common import CommonOps
+
+
+class Logout(CommonOps):
```

Comment:
strange name  - logout is just action, better to rename, something like SuccessfullyLoginPage or like in url LoggedInSuccessfullyPage

Reply:
ok, it is done